File: sample/square.R
stopifnot(flock_run_dir != NULL)
stopifnot(flock_job_dir != NULL)
stopifnot(flock_input_file != NULL)
stopifnot(flock_output_file != NULL)
stopifnot(flock_script_name != NULL)
stopifnot(flock_per_task_state != NULL)
stopifnot(flock_common_state != NULL)

squared = flock_per_task_state ** 2
save(squared, file=flock_output_file)

File: test/py_ss/square.py
  assert(flock_run_dir != None)
  assert(flock_job_dir != None)
  assert(flock_input_file != None)
  assert(flock_output_file != None)
  assert(flock_script_name != None)
  assert(flock_per_task_state != None)
  assert(flock_common_state != None)

  squared = flock_per_task_state ** 2
  save(squared, file=flock_output_file)
  assert(flock_run_dir != None)
  assert(flock_job_dir != None)
  assert(flock_input_file != None)
  assert(flock_output_file != None)
  assert(flock_script_name != None)
  assert(flock_per_task_state != None)
  assert(flock_common_state != None)

  squared = flock_per_task_state ** 2
  save(squared, file=flock_output_file)
